cmds/live.c

```diff
@@ -56,7 +56,7 @@ static bool match_replay_triggers(const char *trigger)
 static void reset_live_opts(struct uftrace_opts *opts)
 {
 	/* this is needed to set display_depth at replay */
-	live_disabled = opts->disabled;
+	live_disabled = !(opts->trace == TRACE_STATE_ON);
```

Comment:
Wouldn't it be `live_disabled = (opts->trace == TRACE_STATE_OFF);` ?

Reply:
Yes, so the initialized `TRACE_STATE_NONE` defaults to ON for this function. I'm fixing it.